### DISCUSSION TITLE
Use frigate for object tracking

### DISCUSSION THREAD

**Discussion by TheSleepySlee** [blakeblackshear/frigate#811] [Ideas]
Hello,

Been using frigate for several months now, working fine.

I was wondering how to use frigate for object tracking: what I mean is being able to, once object is found (like car, person), adjust PTZ camera to always have detected object in the center of the camera.
Is there a way to retrieve coordinates of object found? For example using MQTT.
Wonder what happens if multiple objects found though.

**Comment by blakeblackshear** [comment]
You can use the centroid coordinates and  bounding box from the events mqtt topic. I also have a PTZ, so I will be looking into this at some point.

**Comment by stephengolub** [comment]
Looking at the docs for the MQTT event, you get back the box x,
coordinates: https://blakeblackshear.github.io/frigate/usage/mqtt#frigateevents


Should be able to write an automation that take and calculate a new center or a pan/tilt change.
